tests/test_hist_plot.py

```diff
@@ -169,7 +169,6 @@ def test_plotratio():
         'markersize': 10.,
         'color': 'k',
         'elinewidth': 1,
-        'emarker': '_'
```

Comment:
So this isn't supported anymore? It's a popular request to have caps on the ends of the errorbars.

Reply:
I had to add it manually in the old code (its not in mpl)

Reply:
Sure it is 
```
data_err_opts = {
    'linestyle':'none',
    'marker': '.',
    'markersize': 10.,
    'elinewidth': 1,
    'capsize':4
}
hist.plot.plot1d(kk, overlay='sample', error_opts=data_err_opts)
```
![image](https://user-images.githubusercontent.com/13226500/69264848-be58b400-0b8d-11ea-9339-79d3ed81c518.png)

Reply:
heh ok, I could swear there were arrow caps by default on asymmetric errorbars when I was writing this, maybe it changed?